install/ui/test/ipa_tests.js

```diff
@@ -212,8 +220,14 @@ test("Testing unsuccessful rpc.command().", function() {
 
         equals(data.method, object+'_'+method, "Checking method");
 
+        // By default all rpc calls contain version of API
+        $.extend(options, { 'version': window.ipa_loader.api_version});
+
         same(data.params, [args, options], "Checking parameters");
 
+        // remove api version from options object
+        delete options.version;
```

Comment:
Comment describes the obvious - what it does. But it should describe why it does it which is not clear.